web-components/src/components/menu/Menu.ts

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { customElement, html, LitElement, property, PropertyValues, query } from "lit-element";
+import { MenuItem, MenuItemEvent, MenuItemKeyDownEvent } from "./MenuItem";
+import { Key } from "@/constants";
+import { nanoid } from "nanoid";
+import { SlottedMixin } from "@/mixins/SlottedMixin";
+import { RovingTabIndexMixin } from "@/mixins/RovingTabIndexMixin";
+import { classMap } from "lit-html/directives/class-map";
+import reset from "@/wc_scss/reset.scss";
+import styles from "./scss/module.scss";
+
+type ItemId = Element["id"];
+export const MORE_MENU_ITEM_COPY_ID_PREFIX = "more-menu-item-";
+
+@customElement("md-menu")
+export class Menu extends SlottedMixin(RovingTabIndexMixin(LitElement)) {
+  @property({ type: Boolean }) justified = false;
+  @property({ type: String }) direction = "horizontal"
```

Comment:
in direction="vertical" i have to navigate with left-right arrows instead of up/down

Reply:
Done.